validator/constants.py

```diff
@@ -43,9 +43,9 @@
 with open(os.path.join(os.path.dirname(__file__), 'app_versions.json')) as avs:
     APPROVED_APPLICATIONS = json.load(avs)
 
-# Add the '*' fake max version. Used specifically for WebExtensions, see
-# https://github.com/mozilla/amo-validator/issues/390.
+# Add the '*' fake max version for Desktop and Android.
 APPROVED_APPLICATIONS['1']['versions'].append('*')
+APPROVED_APPLICATIONS['7']['versions'].append('*')
```

Comment:
Oh, I thought you originally meant to patch the real [app_versions.json](https://github.com/mozilla/addons-server/blob/master/src/olympia/devhub/tasks.py#L386) we use on the server. In that case, you'd have to patch it in both places.